doc/api.rst

```diff
@@ -22,6 +22,7 @@ Top-level functions
    full_like
    zeros_like
    ones_like
+   as_variable
```

Comment:
I would move this to the "Advanced API" section at the bottom, next to `Variable` and `IndexVariable`.